ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -169,6 +169,9 @@ peerChurnGovernor tracer deadlineChurnInterval bulkChurnInterval psOverallTimeou
             decrease (targetNumberOfEstablishedBigLedgerPeers base -
                       targetNumberOfActiveBigLedgerPeers base)
             + targetNumberOfActiveBigLedgerPeers base
+        , targetNumberOfBootstrapPeers =
+            decrease (targetNumberOfBootstrapPeers base - targetNumberOfEstablishedPeers base)
```

Comment:
This doesn't look right.
Plain `decrease (targetNumberOfBootstrapPeers base)` should do.
Target is 5, decrease by 20% gives 4.

Reply:
Nice catch :) I just mostly mimicked what `targetNumberRootPeers` was doing